assets/questions/findFirstNonRepeatingCharacter.js

```diff
@@ -38,6 +38,7 @@ globalData.questions['findFirstNonRepeatingCharacter'] = { // eslint-disable-lin
 `
   },
   tasks: [{
+    id: 'findFirstNonrepeatingCharacter',
```

Comment:
findFirstNonRepeatingCharacter, for consistency.

Reply:
Done.